src/components/Profile/Profile.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import './Profile.css';
```

Comment:
Again, you have both. Decide for one and I would recommend using styled-components!